.codeclimate.yml

```diff
@@ -2,16 +2,16 @@ version: "2"         # required to adjust maintainability checks
 checks:
   argument-count:
     config:
-      threshold: 7
+      threshold: 10
```

Comment:
You can also do `disabled: true` or something like that

Reply:
So far this was enough. I actually agree with code climate to some degree that there should not be too many arguments, but with the inference stuff many functions need quite a bit of comments, so I felt this was a good idea